dist/recipe.nsi

```diff
@@ -77,14 +78,40 @@
 Section "Install"
  
   ;Add files
-  SetOutPath "$INSTDIR\generated"  
   SetOutPath "$INSTDIR"
```

Comment:
If we remove the `\generated` we have to make sure that is will be created at some other point.